setup.cfg

```diff
@@ -2,4 +2,16 @@
 universal=1
 
 [flake8]
+exclude=*migrations*
```

Comment:
Do they now fit flake8 / isort rules? I'm used to exclude them, but this come from south days, so Django migrations may behave differently